src/main/java/de/qabel/desktop/ui/contact/ContactController.java

```diff
@@ -274,8 +213,18 @@ private void buildGson() throws EntityNotFoundException, PersistenceException {
     public void update() {
         Platform.runLater(this::loadContacts);
     }
-}
-
 
+    public void openContactMenu(MouseEvent event) {
+        initializeMenu();
+        Platform.runLater(() -> contactMenuController.showMenu(event.getSceneX()
+            + layoutWindow.getScene().getWindow().getX(), event.getSceneY()
+            + layoutWindow.getScene().getWindow().getY() + contactMenu.getHeight()));
```

Comment:
you shouldn't need the `contactMenu.getHeight()` because the PopOver takes care that the arrow ends at your cursor